.config/forest.dic

```diff
@@ -22,6 +22,7 @@ daemonize
 DB/S
 DNS
 Enum
+Ethereum
```

Comment:
Need to bump the word count in line 1 as well?

Reply:
The word count is only there as a performance hint. Given the small size of our dictionary, it doesn't matter at all.

Reply:
Is it fine to get rid of it completely?

Reply:
No, the first line has to contain a number.